demisto_sdk/tests/test_files/Packs/CortexXDR/Playbooks/Invalid_Deprecated_Playbook.yml

```diff
@@ -0,0 +1,644 @@
+id: Cortex XDR Incident Handling
+version: -1
+name: Cortex XDR Incident Handling
+description: "This playbook is triggered by fetching a Palo Alto Networks Cortex XDR
```

Comment:
Do we really need to use test files, or can we use the test utils that @jochman created?
Same for the valid file.

Reply:
I tried to use a playbook that already exists for testing and failed, so I created a new file.